packages/contracts/package.json

```diff
@@ -10,10 +10,13 @@
   ],
   "scripts": {
     "build": "hardhat compile && forge build",
-    "test:hardhat": "hardhat test",
+    "test:tools": "yarn mocha 'hardhat/**/__tests__/*' -R spec", 
```

Comment:
Looks like it must be in separate package

Reply:
https://github.com/eonian-core/farm/pull/240#discussion_r1555205350